### ISSUE TITLE
Let's think about longer LTS versions

### ISSUE THREAD

**Issue by ElectricMaxxx** [ehmprah/sustainable-software-manifesto#3]
to get possibility to longer hardware lifetime. Cause atm several users claim software to be guilty, that they have to by an i.e. smartphone every year. So let's get our software out of it?

**Comment by ehmprah** [comment]
I would disagree insofar as longer LTS version would also mean beneficial updates (which could improve energy usage) would be slower to arrive.
